pkg/client/proxy_backup.go

```diff
@@ -115,7 +116,7 @@ func (c *ProxyClient) BackupRestore(serviceAddress, url, target, volumeName stri
 	if recv.TaskError != nil {
 		var taskErr TaskError
 		if jsonErr := json.Unmarshal(recv.TaskError, &taskErr); jsonErr != nil {
-			err = errors.Wrap(jsonErr, "Cannot unmarshal the restore error, maybe it's not caused by the replica restore failure")
+			err = errors.Wrap(fmt.Errorf(string(recv.TaskError)), "Cannot unmarshal the restore error, maybe it's not caused by the replica restore failure")
```

Comment:
`TaskError` already implements the interface `Error()`. I think you can directly use it `errors.Wrap(recv.TaskError, ......)`.
BTW, should we output `jsonErr` either?

Reply:
We should, even though the decode error is just a cascading error, but it will help us know the complete error context.

Reply:
Updated.